docs/changes.rst

```diff
@@ -16,6 +16,7 @@ Released on August 22nd 2022.
 * Improved reliability of HTML and text files translation.
 * Added support for project level backups.
 * Improved performance and memory usage of translation memory lookups.
+* Added notifications for string removals.
```

Comment:
This should be moved to 4.14.1 as 4.14 has already been released.

Reply:
I have moved this to the correct place now